### PR TITLE
fix(dashboard):  Fix dashboard settings for desks(SD-3110)

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
@ioanpocol can you please mention the ticket number here?

**Comment by ioanpocol** [comment]
SD-3110

File: client/app/scripts/superdesk-workspace/workspace.js [hunk 1]
```diff
@@ -102,12 +102,12 @@
          */
         function getActiveWorkspace() {
             return desks.fetchCurrentDeskId()
-                .then(getActiveWorkspaceId)
+                .then(getActiveWorkspaceId())
```
Comment (plamut) [inline client/app/scripts/superdesk-workspace/workspace.js]: Is this indeed correct? `then()` expects a callback that is invoked on success, and not a promise object itself (but a callback can return the latter).

**Comment by mugurrus** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

**Pull request merged**